src/components/Scatterplot/index.js

```diff
@@ -102,6 +100,7 @@ const ScatterplotComponent = ({
             onClick={onClick}
             zoomMode={ZoomMode.BOTH}
             xScalerFactory={xScalerFactory}
+            zoomAxes={{ x: true, y: true }}
```

Comment:
i guess this means we always have scatter plots being zoomable? Maybe this should be an option like it is in Line Charts

Reply:
Good call -- the `zoomable` property overrides this (as an implementation detail of `InteractionLayer`). I'll make a story to show this, and think about if the API for this should be cleaned up.

Reply:
Oops, I misread it. Turning off zoomable doesn't work -- I'll fix that!